.gitignore

```diff
@@ -0,0 +1,4 @@
+
+
+
+.idea/
```

Comment:
Add this to .gitignore.
Such IDE files should not be commited in project root directory.

Reply:
created one more .gitignore in another folder of project, does it?